xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -284,7 +277,7 @@ bool CAndroidUtils::ProbeResolutions(std::vector<RESOLUTION_INFO>& resolutions)
 
   if (s_hasModeApi)
   {
-    for(RESOLUTION_INFO res : s_res_displayModes)
+    for (RESOLUTION_INFO res : s_res_displayModes)
```

Comment:
is the copy intended?

Reply:
better by reference, I change it, thanks

Reply:
Change to copy as original, thanks